conda_forge_tick/utils.py

```diff
@@ -657,17 +653,26 @@ def _parse_validated_recipes(
                 "build": build_output_data,
             }
         )
-    return {
+
+    parsed_recipes = {
         "about": about_data,
         "build": build_data,
-        "extra": recipe_maintainer_data,
         "package": package_data,
         "requirements": requirements_data,
         "source": source_data,
         "test": None,
         "outputs": output_data,
     }
 
+    return _remove_none_values(parsed_recipes)
+
+
+def _remove_none_values(d):
```

Comment:
I wrote a small utility function to account for this weird edge case. No need to use it but I thought i'd mention in case it was helpful. 

https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/utils.py#L967

The docs are missing which is definitely my bad. The key bit is a pattern like `d.get("key", {}) or {}`. This inserts an empty dict when the key is `None`.